### ISSUE TITLE
Nunjucks templates are being detected as Javascript

### ISSUE THREAD

**Issue by Rowno** [AtomLinter/linter-jshint#20]
I have a `.html` file containing a [Nunjucks][] template:
```
{% extends "templates/base.html" %}

{% block content %}
<article>
    <time>{{ date }}</time>
    {{ contents }}
</article>
{% endblock %}
```

And the first line is being linted as Javascript.

[Nunjucks]: https://mozilla.github.io/nunjucks/

**Comment by steelbrain** [comment]
Fixed long ago
